packages/core/src/decorators/attachment/Attachment.ts

```diff
@@ -44,7 +34,7 @@ export class AttachmentData {
    * Base64-encoded data, when representing arbitrary content inline instead of via links. Optional.
    */
   @IsOptional()
-  @IsBase64()
+  @IsString()
```

Comment:
I'm OK with this change if it fixes the problem but I'm curious about the reason it does not accept unpadded base64. Is an issue with class-validator?

Reply:
I think it only accept plain base64, not any of the variant in between base64 and base64url?